### ISSUE TITLE
10-Minute tutorial uses different versions of Cucumber

### ISSUE THREAD

**Issue by mpkorstanje** [cucumber/docs#367]
Looks like the 10 Minute tutorial is using:

`   "-DarchetypeVersion=4.2.6.1"               \`  for maven, `def cucumberVersion = '4.3.1'` for gradle, and imports from: 4.7:

```
import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;
```

It'd be best to pin the tutorial against 4.2.6.

https://cucumber.io/docs/guides/10-minute-tutorial/

**Comment by mpkorstanje** [comment]
I've released the 4.8.0 version of mvn archetype.

**Comment by mpkorstanje** [comment]
And maybe we can use `gradle init` to turn the maven archetype into a gradle project:

https://stackoverflow.com/a/31263320/3945473
